### ISSUE TITLE
[bug] vcvars_command doesn't allow setting custom winsdk_version

### ISSUE THREAD

**Issue by arsdever** [conan-io/conan#7305]
Hi folks. I have found a very strange behavior when executing `MSBuild.build()` function and trying to use custom specified windows SDK version. The thing is though it is reflected in the command line, it is not used, because another version is set after the version you want to be set. To be more specific, I tried building for UWP and use `<my_sdk_version>` as an SDK version (say it is `10.0.18362.0`). When I look into the command the `Conan` executes it looks like the following

```
...\vcvarsall.bat" amd64 <my_sdk_version> store <another_sdk_version> && ...
```

I dived deeper and found that in this part of code
https://github.com/conan-io/conan/blob/c30284dca57685bf8663dbae4582e43e1a267959/conans/client/tools/win.py#L472-L490
it adds `"store <conans_found_sdk_version>"`, which is being added after custom version is added. Probably I wouldn't have found the issue if the found version was the latest one, but actually it is the oldest one available on the system.

### System information
OS: Windows 10
Compiler: MSVC 14.25
Conan version: observed on 1.23 and 1.27
Python version: 3.8.3 for Conan 1.27 (for 1.23 I don't know)

### Steps to reproduce (Include if Applicable)
Have at least two versions of `WinSDK` installed on the system. Then create a dummy `conanfile.py` and have a `vcvars_command()` function called in the build function with custom `winsdk_version` specified. Ensure, that the specified version is higher than the lowest you have on the system.

**Comment by arsdever** [comment]
Guys any update on this? Is this a known issue? Did I make a mistake?

@memsharded Maybe you can give some information about this?

**Comment by arsdever** [comment]
I see similarities with https://github.com/conan-io/conan/issues/7419

**Comment by memsharded** [comment]
This was fixed by #7584 and will be released in Conan 1.29. Thanks very much for reporting!

**Comment by arsdever** [comment]
Thanks for solving the problem.
